### ISSUE TITLE
Issue with String.localeCompare in Atom Helper

### ISSUE THREAD

**Issue by park9140** [atom/atom#4725]
I am writing a plugin that uses `vinyl-fs`, one of it's node modules uses `String.localeCompare`.  In the new version of atom I am having an issues with this function that does not exist in my local version of node 0.11.14

```bash
~/g/atom-typescript-tools: /Applications/Atom.app/Contents/Frameworks/Atom\ Helper.app/Contents/MacOS/Atom\ Helper
> process.version
'v0.11.14'
> 'asdf'.localeCompare('sd')
fish: Job 1, '/Applications/Atom.app/Contents/Frameworks/Atom\ Helper.app/Contents/MacOS/Atom\ Helper' terminated by signal SIGSEGV (Address boundary error)
drwxr-xr-x jonathanpark:staff typescript-1.4-update ✗
~/g/atom-typescript-tools: node
> process.version
'v0.11.14'
> 'asdf'.localeCompare('sd')
-18
```

As you can see if I open Atom Helper as a node process, run the localeCompare test and it throws a segmentation fault.  In my local node 0.11.14 install it works fine.   I suspect that there is an issue where the localeCompare code utilizes some system library that isn't compiled correctly for my os version.  I am running OSX 10.10 and have not updated to 10.10.1 and the new command line tools yet.

**Comment by park9140** [comment]
@zcbenz, @maxbrunsfeld, Is any more information needed to figure this out?

**Comment by park9140** [comment]
@maxbrunsfeld,  I confirmed this is fixed in `atom-shell@0.20.7`, by `atom/atom-shell/pull/1019`.  Any idea when this new version will be integrated into atom?

**Comment by park9140** [comment]
Seems fixed.
